### DISCUSSION TITLE
Firebase Auth support for non-GMS devices

### DISCUSSION THREAD

**Discussion by orestesgaolin** [firebase/flutterfire#4014] [Feature request]
Firebase team has recently announced that [Firebase Auth SDK version 20.0](https://firebase.google.com/support/release-notes/android#auth_v20-0-0) should be able to work on devices that don't support GMS (Huawei, Amazon etc.). I wonder if this kind of support will work out of the box for Flutter apps? 

Until now, I haven't specified exact dependency for Firebase Auth in my gradle configuraion. It used to look like this for basic app using Firebase:

```
dependencies {
    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
    implementation 'com.android.support:multidex:1.0.3'
    implementation 'com.google.firebase:firebase-analytics:17.5.0'
    implementation 'com.google.firebase:firebase-crashlytics:17.2.2'
}
```

I haven't tried it yet, but maybe someone has already succeeded with running Firebase Auth on non-GMS devices?

[Article](https://firebase.googleblog.com/2020/10/firebase-auth-android-non-gms.html)

**Comment by Salakar** [comment]
Please could you try my suggestion here: https://github.com/FirebaseExtended/flutterfire/issues/4004#issuecomment-721395427

**Reply by orestesgaolin** [reply]
Thanks, I added my findings [there](https://github.com/FirebaseExtended/flutterfire/issues/4004#issuecomment-721644139). Closing this in favor of the #4004
